demo/src/app/components/datepicker/demos/basic/datepicker-basic.ts

```diff
@@ -9,8 +9,13 @@ const now = new Date();
 export class NgbdDatepickerBasic {
 
   model;
+  disabled = false;
 
   selectToday() {
     this.model = {year: now.getFullYear(), month: now.getMonth(), day: now.getDate()};
   }
+
+  toggleDisabled() {
```

Comment:
We could probably in-line this in a template. Having said this I'm not sure it makes sense to demonstrate it as part of the very basic demo. If I recall properly we want to have to the basic demo as simple as possible (that is, use default values for all props and provide only ones that are must-have for the most rudimentary demo).